e2e/benchmarks/local-benchmark/index.html

```diff
@@ -110,11 +103,70 @@ <h2>TensorFlow.js Model Benchmark</h2>
     let warmupTimes = 1;
     // Default do not run any task.
     let task = '';
+
+    function loadScript(url, callback) {
+      var script = document.createElement("script")
+      script.type = "text/javascript";
+
+      if (script.readyState) { //IE
+        script.onreadystatechange = function() {
+          if (script.readyState == "loaded" ||
+          script.readyState == "complete") {
+            script.onreadystatechange = null;
+            callback();
+          }
+        };
+      } else { //Others
+        script.onload = function() {
+          callback();
+        };
+      }
+
+      script.src = url;
+      document.getElementsByTagName("head")[0].appendChild(script);
+    }
+
+    var unpkgUrlsIndex = 0;
```

Comment:
We need to support the hybrid mode, that is some urls are from unpkg, while the others are from self-built. Then we can have a parameter like "localDist=core,cpu,webgpu" (customtfjs is not a good name for me) to use the ones we build from source code.